aura.go

```diff
@@ -94,14 +97,14 @@ func runAura(cmd *Command, args []string) {
 			fmt.Println("Must specify entity type and name")
 			os.Exit(2)
 		}*/
-		ErrorAndExit("force aura create not yet implemented")
+		util.ErrorAndExit("force aura create not yet implemented")
 
 	case "delete":
 		runDeleteAura()
 	case "list":
 		bundles, err := force.GetAuraBundlesList()
 		if err != nil {
-			ErrorAndExit(err.Error())
+			util.ErrorAndExit(err.Error())
```

Comment:
I think that's a fair idea, but let's leave that for another PR.  Just getting it until the util package is a good start, I think! :)